doc/classes/TextureLayered.xml

```diff
@@ -1,10 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextureLayered" inherits="Texture" version="4.0">
 	<brief_description>
-		Base class for 3D texture types.
+		Base class for texture types which represent an array of [Image]s of the same size and format.
 	</brief_description>
 	<description>
-		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types. Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		Base class for [Texture2DArray], [Cubemap] and [CubemapArray]. Cannot be used directly, but contains all the functions necessary for accessing the derived resource types.
+		Data is set on a per-layer basis. For [Texture2DArray]s, the layer specifies the array layer.
+		All images need to have the same width, height and number of mipmap levels.
+		A [TextureLayered] can be loaded with [code]method ResourceFormatLoaderTextureLayered.load[/code].		
+		To create a [TextureLayered] texture, an image file needs to be "sliced". Internally, Godot uses the thirdparty basis_universal library by Binomial LLC for e.g. creating and reading [Texture2DArray], [Cubemap] and [CubemapArray] files.
+		Their file extension is normally [code].basis[/code]. Godot adjust this to its needs:
+		[codeblock]
+		- the Texture2DArray becomes a ".tex2darr"
+		- the Cubemap becomes a ".cube"
+		- the CubemapArray becomes a ".cubearr" 
+		[/codeblock]
+		The [code]*.basis[/code] file remembers how to apply the slicing.
+		basis_universal can be accessed via the Godot Editor (during the image file import) and the basisu commandline tool (see file [code]thirdparty/basis_universal/basisu_tool.cpp[/code] regarding [code]-tex_type 2darray[/code]). With this library you combine several image files (e.g. png) to one *.basis file. This file will be read and sliced when you load it (see [code]basisu_backend.compute_slice_crcs[/code]).
```

Comment:
This is probably an explanation that the end-user doesn't want or need. I feel like it is more likely to confuse users than it is inform them.

Reply:
hm... throwing it away feels a bit of a waste.  
How about adding a bold line between the last 3 lines and the codeblock stating **Engine Excursion** or sth. along the line?

I understand that most users don't want too much detail, but there are some advanced users (as you mentioned) which might benefit from this. Marking a text with "this is also information, but not for Godot users but engine developers"? I don't really want to exclude either of those 2 target groups

What's your thought on this?

Reply:
The information about the file extension is unnecessary. A step by step guide on how to create these resources, and how to access them in a uniform, would be more valuable.

Reply:
Overall this is some rather low level shader stuff only relevant to experienced users. So getting into the nitty gritty is appreciated.

Reply:
cut, except the file extension -> will remove them too, if strongly desired